packages/eslint-config-eventbrite/rules/esnext.js

```diff
@@ -6,12 +6,23 @@ module.exports = {
         ecmaFeatures: {
         },
     },
+    plugins: ['babel'],
 
     rules: {
         // disallow await inside of loops
         // https://eslint.org/docs/rules/no-await-in-loop
         'no-await-in-loop': 'error',
 
+        // allow `this` keywords outside of classes or class-like objects
+        // so that `babel/no-invalid-this` can disallow (see below)
+        // http://eslint.org/docs/rules/no-invalid-this
+        'no-invalid-this': 'off',
```

Comment:
I think this helps it be more explicit and ties to directly to why. It also last less risk of it getting into a halfway state where it is off, but should be on.

Reply:
Very good point. Thanks the review